pkg/kubelet/checkpointmanager/checkpoint_manager.go

```diff
@@ -85,9 +85,6 @@ func (manager *impl) GetCheckpoint(checkpointKey string, checkpoint Checkpoint)
 		return err
 	}
 	err = checkpoint.UnmarshalCheckpoint(blob)
-	if err == nil {
-		err = checkpoint.VerifyChecksum()
```

Comment:
@bart0sh: can you add a commit message explaining the "what" and "why" of this commit?

Reply:
as we moved to another approach this is not needed anymore. checksum is always verified. see details in the types.go

Reply:
@bart0sh I'm not quite sure what you mean? This is in the generic library code, and Patrick made changes to the devicemanager and the cpumanager to adhere to this API. I don't think we can just remove this because of that.

Reply:
Sorry for the confusion. This is an old code and it's not in this PR anymore.

Reply:
Maybe you forgot to remove it? I am commenting on this from the "Files Changed" tab of the PR...

Reply:
my bad. removed. Thank you for pointing out.